pykern/pkio.py

```diff
@@ -125,6 +127,20 @@ def has_file_extension(filename, to_check):
     return e in to_check
 
 
+def is_pure_text(filename, chunk_size=512):
+    """Uses pykern.util.is_pure_text to guess whether file is pure text
+
+    Args:
+        filename (str|py.path.local): file to check
+
+    Returns:
+        bool: True if file is likely pure text, false if likely binary
+    """
+    with open(filename, "rb") as f:
+        b = f.read(chunk_size + _POSSIBLE_VALID_CHAR_BYTES)
```

Comment:
no need for temporary variable. No need for _POSSIBLE_VALID_CHAR_BYTES. Boundary is arbitrary. See discussion below.